src/qol/stereo-default.s

```diff
@@ -0,0 +1,76 @@
+/*  Sets the default speaker mode to Stereo
+*/
+.org DefaultStereoFlagPointer
+.area 4
+    .dw     DefaultStereoFlag
+.endarea
+
+.org 0809E8BAh
+; Modifying OAM setup in FileSelectInit()
+.area 4
+    bl      @FileSelectInitHighjack_SetupOam
+.endarea
+
+.org 0809E8DEh
+; Highjacks FileSelectInit(), enables stereo automatically
+.area 4
+    bl      @FileSelectInitHighjack_CheckEnableStereo
+.endarea
+
+.org 0809F670h
+; Highjacks FileSelectSetupOam()
+.area 4
+    bl      @FileSelectSetupOamHighjack
+.endarea
+
+.autoregion
+DefaultStereoFlag:
+    .db     00
+    .align 2
+@FileSelectInitHighjack_SetupOam:
```

Comment:
comment explanation on what r0 does in the context of fileselectsetupoam would be nice

Reply:
you're thinking of the wrong function, this function doesnt have that comment.
This function returns r0 and as far as i can see is then used later as an argument to a future function after the hijack returns

Reply:
oh you made me realize i misread the last lines as
```arm
   mov     r**0**, #0AAh
@@mono:
    mov     r0, #09h
```

whats the use in setting r1 to that value if it gets immediately overwritten after t he hijack to 0x66?